src/lib/components/core/popups/popups/Feedback.svelte

```diff
@@ -0,0 +1,105 @@
+<script lang="ts">
+import Warning from "$components/core/popups/popups/Warning.svelte";
+import Button from "$components/ui/Button.svelte";
+import TextArea from "$components/ui/TextArea.svelte";
+import TextInput from "$components/ui/TextInput.svelte";
+import { type PopupState, popups } from "$state/popup.svelte";
+import * as Sentry from "@sentry/browser";
+import { getContext } from "svelte";
+import { _ } from "svelte-i18n";
+import type { Writable } from "svelte/store";
+
+let comments = "";
+let senderName = "";
+let senderEmail = "";
+const popupState = getContext<Writable<PopupState>>("state");
+
+function cancel() {
+	popups.close($popupState.id, false);
+}
+
+async function save() {
+	const eventId = Sentry.captureMessage("User Feedback");
+	const userFeedback = {
+		event_id: eventId,
+		name: senderName,
+		email: senderEmail,
+		comments: comments,
+	};
+	Sentry.captureUserFeedback(userFeedback);
+
+	await popups.open({
+		component: Warning,
+		data: {
+			title: "FEEDBACK",
+			message: $_("FEEDBACK_SENT"),
```

Comment:
this seems kinda overkill as the Warning popup already runs the translation function, also it will break live translations, because the value won't be updated